config/locales/doorkeeper.en.yml

```diff
@@ -1,152 +1,140 @@
-        unsupported_grant_type: 'The authorization grant type is not supported by the authorization server.'
-
+          request_not_authorized: Request need to be authorized. Required parameter for authorizing request is missing or invalid.
+          unknown: The request is missing a required parameter, includes an unsupported parameter value, or is otherwise malformed.
+        invalid_scope: The requested scope is invalid, unknown, or malformed.
         invalid_token:
-          revoked: "The access token was revoked"
-          expired: "The access token expired"
-          unknown: "The access token is invalid"
+          expired: The access token expired
+          revoked: The access token was revoked
+          unknown: The access token is invalid
+        resource_owner_authenticator_not_configured: Resource Owner find failed due to Doorkeeper.configure.resource_owner_authenticator being unconfigured.
         revoke:
-          unauthorized: "You are not authorized to revoke this token"
-
+          unauthorized: You are not authorized to revoke this token
+        server_error: The authorization server encountered an unexpected condition which prevented it from fulfilling the request.
+        temporarily_unavailable: The authorization server is currently unable to handle the request due to a temporary overloading or maintenance of the server.
+        unauthorized_client: The client is not authorized to perform this request using this method.
+        unsupported_grant_type: The authorization grant type is not supported by the authorization server.
+        unsupported_response_mode: The authorization server does not support this response mode.
+        unsupported_response_type: The authorization server does not support this response type.
     flash:
       applications:
         create:
-          notice: 'Application created.'
+          notice: Application created.
         destroy:
-          notice: 'Application deleted.'
+          notice: Application deleted.
         update:
-          notice: 'Application updated.'
+          notice: Application updated.
       authorized_applications:
         destroy:
-          notice: 'Application revoked.'
-
+          notice: Application revoked.
     layouts:
       admin:
-        title: 'Doorkeeper'
         nav:
-          oauth2_provider: 'OAuth2 Provider'
-          applications: 'Applications'
-          home: 'Home'
+          applications: Applications
+          home: Home
+          oauth2_provider: OAuth2 Provider
+        title: Doorkeeper
       application:
-        title: 'OAuth authorization required'
+        title: OAuth authorization required
+    pre_authorization:
+      status: Pre-authorization
+    scopes:
+      public: Access public profile data and published articles
+      read_articles: Access all articles including drafts
+      write_articles: Access to creating and updating articles
```

Comment:
here we only add these three custom keys, so we might as well keep them separate and use the default locale file inherited from the gem